lms/djangoapps/verify_student/tests/test_models.py

```diff
@@ -330,6 +330,27 @@ def test_initial_verification_for_user(self):
             self.assertIsNotNone(fourth_result)
             self.assertEqual(fourth_result, first_result)
 
+    def test_retire_user(self):
+        user = UserFactory.create()
+        user.profile.name = u"Enrique"
+
+        attempt = SoftwareSecurePhotoVerification(user=user)
+        attempt.mark_ready()
+        attempt.status = "submitted"
+        attempt.photo_id_image_url = "https://example.com/test/image/img.jpg"
+        attempt.face_image_url = "https://example.com/test/face/img.jpg"
+        attempt.approve()
+
+        # Before Delete
+        assert_equals(attempt.name, user.profile.name)
+        assert_equals(attempt.photo_id_image_url, 'https://example.com/test/image/img.jpg')
+        assert_equals(attempt.face_image_url, 'https://example.com/test/face/img.jpg')
+
+        # Attempt
+        self.assertTrue(attempt.delete_by_user_value())
```

Comment:
Sorry too say that because `delete_by_user_value` is a class method and needs the user value specified, this should be `PhotoVerification.delete_by_user_value(user.id, "user_id")`.

Reply:
Also, it's "user", not "user_id", right?